test/specs/runLwcTests.e2e.ts

```diff
@@ -12,554 +12,550 @@ import path from 'path';
+    utilities.log(`${testSetup.testSuiteSuffixName} - Run All Tests on a LWC via the Test Sidebar`);
+    const workbench = await utilities.getWorkbench();
+    const testingView = await workbench.getActivityBar().getViewControl('Testing');
+
+    // Open the Test Sidebar
+    const testingSideBarView = await testingView?.openView();
+    await expect(testingSideBarView).toBeInstanceOf(SideBarView);
+
+    // Click the run test button that is shown to the right when you hover a test class name on the Test sidebar
+    const lwcTestsSection = await utilities.getTestsSection(workbench, 'LWC TESTS');
+    const lwcTestItem = (await lwcTestsSection.findItem('lwc1')) as TreeItem;
+    await lwcTestItem.select();
+    const runTestsAction = await lwcTestItem.getActionButton(
+      'SFDX: Run Lightning Web Component Test File'
+    );
+    if (!runTestsAction) {
+      fail('Could not find run tests action button');
+    }
+    await runTestsAction.elem.click();
+
+    // Verify test results are listed on the terminal
+    // Also verify that all tests pass
+    const terminalText = await utilities.getTerminalViewText(
+      workbench,
+      utilities.Duration.seconds(15)
+    );
+    await expect(terminalText).not.toBeUndefined();
+    await expect(terminalText).toContain(
+      `PASS ${path.join(
+        'force-app',
+        'main',
+        'default',
+        'lwc',
+        'lwc1',
+        '__tests__',
+        'lwc1.test.js'
+      )}`
+    );
+    await expect(terminalText).toContain('Test Suites: 1 passed, 1 total');
+    await expect(terminalText).toContain('Tests:       2 passed, 2 total');
+    await expect(terminalText).toContain('Snapshots:   0 total');
+    await expect(terminalText).toContain(
+      `Ran all test suites within paths "${path.join(
+        projectFolderPath,
+        'force-app',
+        'main',
+        'default',
+        'lwc',
+        'lwc1',
+        '__tests__',
+        'lwc1.test.js'
+      )}`
+    );
+
+    // Verify the tests that are passing are labeled with a green dot on the Test sidebar
+    const icon = await (await lwcTestItem.elem).$('.custom-view-tree-node-item-icon');
+    const iconStyle = await icon.getAttribute('style');
+    // Try/catch used to get around arbitrary flaky failure on Ubuntu in remote
+    try {
+      await expect(iconStyle).toContain('testPass');
+    } catch {
+      utilities.log('ERROR: icon did not turn green after test successfully ran');
+    }
+  });
+
+  step('Run Single Test via the Test Sidebar', async () => {
+    utilities.log(`${testSetup.testSuiteSuffixName} - Run Single Test via the Test Sidebar`);
+    const workbench = await utilities.getWorkbench();
+    const testingView = await workbench.getActivityBar().getViewControl('Testing');
+
+    // Open the Test Sidebar
+    const testingSideBarView = await testingView?.openView();
+    await expect(testingSideBarView).toBeInstanceOf(SideBarView);
```

Comment:
Seems that if testingView is undefined, the rest of the step is moot.
With not undefined expect the step will fail for the correct reason if the testing activity is not present.
``` suggestion
    const testingView = await workbench.getActivityBar().getViewControl('Testing');
    await expect(testingView).not.toBeUndefined();

    // Open the Test Sidebar
    const testingSideBarView = await testingView?.openView();
    await expect(testingSideBarView).toBeInstanceOf(SideBarView);
```